### PR TITLE
ItemAdapter.get_field_names_from_class

### PR DESCRIPTION
**Pull request by kmike**
hey! This PR adds a new ItemAdapter classmethod: get_field_names_from_class - similar to get_field_names, but accepts item class, not an item itself. It's motivated by https://github.com/scrapinghub/web-poet/pull/53#discussion_r934313210.

### CODE REVIEW

File: tests/test_adapter_dataclasses.py [hunk 1]
```diff
@@ -43,10 +43,13 @@ def test_module_import_error(self):
             from itemadapter.adapter import DataclassAdapter
 
             self.assertFalse(DataclassAdapter.is_item(DataClassItem(name="asdf", value=1234)))
-            with self.assertRaises(RuntimeError, msg="attr module is not available"):
+            with self.assertRaises(RuntimeError, msg="dataclasses module is not available"):
```
Comment (kmike) [inline tests/test_adapter_dataclasses.py:46]: I'm not sure why it's been passing. Probably, assertRaises doesn't work as intended here.

Comment (Gallaecio) [reply tests/test_adapter_dataclasses.py:46]: I believe that is the message that shows if the assert fails. We can use `assertRaisesRegex` if the goal is for the assert of fail if the exception message does not match this one.

**Review by Gallaecio — APPROVED** [review]
Nice!

**Review by elacuesta — APPROVED** [review]
Looks good @kmike, thanks! Should I release now with this or is there anything else you need?

**Comment by kmike** [comment]
Thanks for merging it @elacuesta! 
Yes, a release would be perfect; I don't have anything else in mind.

**Comment by elacuesta** [comment]
Released https://github.com/scrapy/itemadapter/releases/tag/v0.7.0

**Comment by kmike** [comment]
Awesome, thanks @elacuesta!

**Pull request merged**